src/main/c/sslcontext.c

```diff
@@ -1396,70 +1396,13 @@ TCN_IMPLEMENT_CALL(void, SSLContext, setSessionTicketKeys0)(TCN_STDARGS, jlong c
 
 /* OpenSSL end */
 
-/*
- * Adapted from Android:
- * https://android.googlesource.com/platform/external/openssl/+/master/patches/0003-jsse.patch
- */
-const char* SSL_CIPHER_authentication_method(const SSL_CIPHER* cipher){
```

Comment:
@nmittler are you sure "SSL_authentication_method " does the same as the method here ?